src/dotnet/Fable.Compiler/CLI/ProjectCracker.fs

```diff
@@ -314,7 +314,7 @@ let createFableDir rootDir =
     fableDir
 
 let copyDirIfDoesNotExist (source: string) (target: string) =
-    if isDirectoryEmpty target then
+    if Directory.Exists source && isDirectoryEmpty target then
```

Comment:
Why did you have to add this, because non-existent `fable-core`? Maybe we can take two birds with one stone and fix this (and the comment above) by passing `--fable-core` through the CLI?

Reply:
Yeah, it doesn't hurt to check if source exists, it's just a check.

Reply:
Hmm, it may be good to get the error if the source doesn't exist, I can't think right now in which situations missing dependency's sources (either fable-core or another Nuget package) are not an issue 🤔

Reply:
If we want a "fable-core is missing" error, perhaps an explicit check and friendly error message would be better than just throwing.